CMakeLists.txt

```diff
@@ -15,7 +15,7 @@ option(XSIMD_REFACTORING ON)
 
 set(XSIMD_INCLUDE_DIR ${CMAKE_CURRENT_SOURCE_DIR}/include)
 
-# Versionning
+# Versioning
 # ===========
```

Comment:
You need to remove one `=` below too ;-)

Reply:
Good point. I'll do that